src/Behat/Cores/CoreInterface.php

```diff
@@ -278,8 +280,10 @@ public function getDbLogMessages(int $scenario_start_time, array $severities = [
    *   Entity type.
    * @param array $entities_ids
    *   Entity id list.
+   * @param bool $reset_cache
+   *   Wether the entity caches should be reset before loading them
```

Comment:
Could you add a comment explaining why reset cache is default to FALSE, with an specific use case that justifies it.